conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ParametersHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.runtime;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Multimap;
+import com.palantir.conjure.java.undertow.lib.Parameters;
+import com.palantir.logsafe.Preconditions;
+import io.undertow.server.HttpHandler;
+import io.undertow.server.HttpServerExchange;
+
+/**
+ * {@link HttpHandler} that attaches to the exchange a safe and an unsafe parameters MultiMap such that a subsequent
+ * handler can attach parameters to the exchange using {@link Parameters} utility methods.
+ */
+public final class ParametersHandler implements HttpHandler {
```

Comment:
If we keep this, it should be package private

Reply:
See below, I am fine with removing this class.